app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
Please add `:confirmable` to `devise` so the app is able to confirm credentials.

Reply:
Thank you again for your reviews. I have added this `:confirmable` attribute in the users model.